src/app/components/CardCarousel/CardCarousel.module.css

```diff
@@ -0,0 +1,248 @@
+.carouselWrapper {
+  position: relative;
+  width: 100%;
+  padding: 20px;
+}
+
+.slider {
+  overflow: hidden;
+}
+
+.slide {
+  background-color: #f5f5f5;
+  border-radius: 12px;
+  text-align: center;
+  width: 240px;
+  height: 320px;
+  min-width: 240px;
+  max-width: 240px;
+  min-height: 320px;
+  max-height: 320px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: hidden;
+}
+
+.videoPlaceholder {
+  background-color: #110404;
+  border-radius: 8px;
+  height: 140px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 12px;
+  overflow: hidden;
+  position: relative;
+  aspect-ratio: 16 / 9; /* Same ratio as image */
+}
+.videoPlaceholder img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+  object-position: center top;
+}
+
+.playIcon {
+  font-size: 32px;
+  color: #555;
+}
+
+.comingSoon {
+  font-size: 14px;
+  color: #777;
+}
+
+.info {
+  text-align: left;
+  margin-left: 10px;
+  margin-right: 5px;
+}
+
+.label {
+  font-size: 12px;
+  color: #888;
+  text-transform: uppercase;
+}
+
+.title {
+  font-size: 16px;
+  font-weight: 600;
+  margin: 6px 0;
+  color: #333;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: normal;
+  display: block;
+  line-height: 1.4em;
+  max-height: 2.8em;
+}
+
+.meta {
+  font-size: 12px;
+  color: #666;
+}
+
+.navButton {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  background: white;
+  border: none;
+  font-size: 24px;
+  padding: 8px 12px;
+  border-radius: 50%;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.15);
+  cursor: pointer;
+  z-index: 10;
+}
+
+.left {
+  left: -10px;
+}
+
+.right {
+  right: -10px;
+}
+
+.topRow {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin-bottom: 8px;
+}
+
+.viewAllLink {
+  font-size: 14px;
+  color: #0070f3;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.mediaWrapper {
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 60%;
+  margin-bottom: 12px;
+}
+.cardLink {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.media {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+@media (max-width: 1440px) {
+  .media {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+}
+@media (max-width: 1024px) {
+  .media {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+}
+@media (max-width: 768px) {
+  .media {
+    width: 100%;
+    height: 100%;
+    object-fit: fill;
+  }
+}
+
+@media (max-width: 320px) {
+  .carouselWrapper {
+    position: relative;
+    width: 100%;
+    padding: 0px;
+  }
+  .media {
+    width: 100%;
+    height: 100%;
+    object-fit: fill;
+  }
+  .slide {
+    background-color: #f5f5f5;
```

Comment:
It would be better if you create a theme file and reference colors from that theme file